src/routes/api/profile.js

```diff
@@ -0,0 +1,18 @@
+import { Router } from 'express';
+import ProfileController from '../../controllers/profileController';
+import middleware from '../../middlewares';
+
+const { Authenticate, validate } = middleware;
+
+const router = Router();
+
+router.route('/')
+  .post(Authenticate.verifyToken, ProfileController.upsertProfile)
+  .get(Authenticate.verifyToken, Authenticate.verifyTravelAdmin, ProfileController.getAllProfiles);
+router.route('/:userId')
+  .get(Authenticate.verifyToken, validate('checkUserId'), Authenticate.verifyOwnership, ProfileController.getProfile)
```

Comment:
This route should be available not only to the owner of the profile but also to the manager of the profile owner and the super admin.

Reply:
Done.